pkg/provider/helm/values.go

```diff
@@ -158,26 +169,57 @@ func (g *HelmValuesGenerator) GenerateValues() (map[string]any, error) {
 		"enabled": false,
 	}
 
+	cses, err := getOrCreateNestedMap(identities, "clusterStaticEntries")
+	if err != nil {
+		return nil, fmt.Errorf("failed to get clusterStaticEntries map from identities: %w", err)
+	}
+
 	filter := &datasourcepb.ListAPBindingsRequest_Filter{TrustZoneName: &g.trustZone.Name}
 	bindings, err := g.source.ListAPBindings(filter)
 	if err != nil {
 		return nil, fmt.Errorf("failed to list attestation policy bindings: %w", err)
 	}
 
-	// Adds the attestation policies as ClusterSPIFFEID CRs to be reconciled by the spire-controller-manager.
+	staticAP := false
```

Comment:
This bool flips to true if any of the collection `bindings` is a static one? Maybe we can rename to something more descriptive & functional, like `needsClusterStaticEntryForAgents`  or similar?